tests/integration/test_script_runner.py

```diff
@@ -121,6 +142,16 @@ def test_docs(test_configuration, tmp_path, pytest_parsed_arguments):
         )
         script_path = os.path.join(tmp_path, "test_script.py")
         shutil.copyfile(script_source, script_path)
+
+        # Util Script
+        if test_configuration.get("util_script") is not None:
+            script_source = os.path.join(
+                test_configuration.get("base_dir"),
+                test_configuration.get("util_script"),
+            )
+            script_path = os.path.join(tmp_path, "util.py")
+            shutil.copyfile(script_source, script_path)
+
```

Comment:
I was having some trouble getting `util.py` to import correctly. Decided on a (potentially) temporary solution where we copy the file to each testing directory